DomainModelCore/ViaCanteen.Ordering.CoreDomain/Payment/PymentTest.fs

```diff
@@ -0,0 +1,33 @@
+module PaymentTest
+
+open System
+open Payment 
+open ViaCard
+open CreditCard
+open MobilePay
+
+
+
+// Test ViaCard
```

Comment:
maybe remove the comments, they are redundant in this case, the function declarations are verbose